flocker/restapi/_infrastructure.py

```diff
@@ -58,16 +74,10 @@ def _logging(original):
     """
     @wraps(original)
     def logger(self, request, **routeArguments):
-        try:
-            logger = self.logger
-        except AttributeError:
-            logger = _logger
-        else:
-            if logger is None:
-                logger = _logger
+        logger = _get_logger(self)
 
-        path = repr(request.path).decode("ascii")
-        action = REQUEST(logger, request_path=path)
+        action = REQUEST(logger, request_path=request.path,
+                         method=request.method)
```

Comment:
No multi-field serializers so far, unfortunately. And given only one call site...